src/Avalonia.Diagnostics/Diagnostics/ViewModels/AvaloniaPropertyViewModel.cs

```diff
@@ -11,6 +11,12 @@ internal class AvaloniaPropertyViewModel : PropertyViewModel
         private string _priority;
         private string _group;
         private readonly Type _propertyType;
+        private enum PinnedStatus
+        {
+            Pinned,
+            Unpinned
+        }
+        private PinnedStatus _isPinned;
```

Comment:
Why define this as an enum? The "Is" name prefix implies it is a bool. If you keep it as a bool you can avoid the converter entirely, correct?